### ISSUE TITLE
Menu's on-hover background height feels small

### ISSUE THREAD

**Issue by pankgeorg** [JuliaDocs/Documenter.jl#1899]
![image](https://user-images.githubusercontent.com/8681967/183393031-59cbd912-068e-46a5-840a-7b843070221b.png)

**Comment by pfitzseb** [comment]
Could also do with a bit of horizontal padding.

**Comment by Hetarth02** [comment]
After increasing `padding` from `0` to `0.3rem` it looks much better.

| Before | ![image](https://user-images.githubusercontent.com/80420115/222243134-9e413fae-e13a-4460-80bb-8149fa2559ba.png) |
|---|---|
|  After  | ![image](https://user-images.githubusercontent.com/80420115/222241777-e60366bc-ddcc-4a47-bbde-e09382ce637a.png)  |

**Comment by Hetarth02** [comment]
@mortenpi Can you take a look at this too?

**Comment by mortenpi** [comment]
I feel that, rather than adding padding, we should expand the gray highlight to cover the gap between the list items:

![image](https://user-images.githubusercontent.com/147757/224578427-df5f32ed-5f81-46df-8a89-92b6a43fe74c.png)

Otherwise it will always "feel small" relative to the list item itself. But I think for that we need to add background color to the `li`, rather than the `a`.

> Could also do with a bit of horizontal padding.

@pfitzseb Where do you actually mean? In front of the list bullet?

**Comment by pfitzseb** [comment]
Yeah, just add a bit of horizontal and vertical padding to the `li` (and add the hover effect for that instead of the anchor):
![image](https://user-images.githubusercontent.com/6735977/224670327-fab1e2b0-afc0-450c-829d-1e5b2dd3383a.png)

**Comment by Hetarth02** [comment]
@pfitzseb I cannot understand what you are saying. Are you implying to add the hover effect to `li` tag after adding padding to the it?

If so, doing that results in this on hover,
![image](https://user-images.githubusercontent.com/80420115/226102422-db004ff0-fd9a-4b2f-8e82-d50701dff899.png)

**Comment by mortenpi** [comment]
> Are you implying to add the hover effect to `li` tag after adding padding to the it?

The idea is to fill the gap between the items, without (necessarily) changing the spacing between the items.

![image](https://user-images.githubusercontent.com/147757/227097529-69c8b0d2-6275-4957-92c7-3fd76e76d0d5.png)

That does require having the `:hover` effect on the `li`, yes. A "problem" we have right now is that we're doing "padding" with margins in the `<li>`s. But it should be fine to switch to `padding-*` there.

Additionally, we may want to adjust the horizontal padding as well.
